pubspec.yaml

```
@@ -1,16 +1,16 @@
 name: bazel_worker
-version: 0.1.25
+version: 1.0.0-nullsafety.0
 
 description: Tools for creating a bazel persistent worker.
 homepage: https://github.com/dart-lang/bazel_worker
 
 environment:
-  sdk: '>=2.3.0 <3.0.0'
+  sdk: '>=2.12.0-0 <3.0.0'
 
 dependencies:
-  async: '>1.9.0 <3.0.0'
-  pedantic: ^1.8.0
-  protobuf: '>=0.14.4 <2.0.0'
+  async: '>=2.5.0-nullsafety.3 <3.0.0'
+  pedantic: ^1.10.0-nullsafety.3
+  protobuf: '>=2.0.0-nullsafety.0 <3.0.0'
```

nit, we don't need to indicate the versions here

```suggestion
  async: '>=2.5.0-nullsafety <3.0.0'
  pedantic: ^1.10.0-nullsafety
  protobuf: '>=2.0.0-nullsafety <3.0.0'
```

---

Done.